GOVERNANCE.md

```diff
@@ -0,0 +1,221 @@
+# cert-manager Governance
+
+This document defines project governance for the cert-manager project. Its
+purpose is to describe how decisions are made on the project and how anyone can
+influence these decisions. We have six levels of responsibility, each one
+building on the previous:
+
+- Contributor,
+- GitHub Member,
+- Reviewer,
+- Approver,
+- Maintainer,
+- Admin.
+
+## Contributors
+
+cert-manager is for everyone. Anyone can become a cert-manager contributor
+simply by contributing to the project, whether through code, documentation, blog
+posts, community management, or other means. As with all cert-manager community
+members, contributors are expected to follow the [cert-manager Code of
+Conduct][coc].
+
+All contributions to cert-manager code, documentation, or other components in
+the cert-manager GitHub org must follow the guidelines in [the contributing
+page][contrib]. Whether these contributions are merged into the project is the
+prerogative of the reviewers, approvers and/or maintainers.
+
+## GitHub Members
+
+GitHub Members are active contributors to the cert-manager project.
```

Comment:
Excellent point, I think we kind of forgot to include all of the project of the org, e.g., how one can become an approver or reviewer of a certain project instead of the whole org.